src/PointwiseFunctions/GeneralRelativity/WeylElectric.hpp

```diff
@@ -41,6 +41,29 @@ void weyl_electric(
     const tnsr::II<DataType, SpatialDim, Frame>&
         inverse_spatial_metric) noexcept;
 
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes a scalar version of the electric part of the Weyl tensor
```

Comment:
`a scalar version...` -> `the scalar \f$E_{ij} E^{ij}\f$ from the electric part of the Weyl tensor \f$E_{ij}\f$.